<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-7NMSRQD" failures="3" tests="8" name="LoginTests" time="90.281" errors="0" timestamp="2025-06-25T11:50:07 IST">
  <testcase classname="tests.LoginTests" name="validLogin" time="43.597"/>
  <testcase classname="tests.LoginTests" name="validLogin" time="3.614">
    <failure type="java.lang.AssertionError" message="Test failed due to Invalid Password.">
      <![CDATA[java.lang.AssertionError: Test failed due to Invalid Password.
at tests.LoginTests.validLogin(LoginTests.java:151)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 26 stack frames]]>
    </failure>
  </testcase> <!-- validLogin -->
  <testcase classname="tests.LoginTests" name="validLogin" time="1.904">
    <failure type="java.lang.AssertionError" message="Test failed due to Invalid Username/Account.">
      <![CDATA[java.lang.AssertionError: Test failed due to Invalid Username/Account.
at tests.LoginTests.validLogin(LoginTests.java:137)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 26 stack frames]]>
    </failure>
  </testcase> <!-- validLogin -->
  <testcase classname="tests.LoginTests" name="validLogin" time="1.256"/>
  <testcase classname="tests.LoginTests" name="invalidLogin" time="8.122">
    <failure type="java.lang.AssertionError" message="Test failed due to Valid Credentials.">
      <![CDATA[java.lang.AssertionError: Test failed due to Valid Credentials.
at tests.LoginTests.invalidLogin(LoginTests.java:268)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 26 stack frames]]>
    </failure>
  </testcase> <!-- invalidLogin -->
  <testcase classname="tests.LoginTests" name="invalidLogin" time="1.92"/>
  <testcase classname="tests.LoginTests" name="invalidLogin" time="1.524"/>
  <testcase classname="tests.LoginTests" name="invalidLogin" time="2.933"/>
</testsuite> <!-- LoginTests -->
